.github/workflows/push.yaml

```diff
@@ -0,0 +1,30 @@
+name: Push
+on:
+  push:
+    branches:
+    - main
+jobs:
+  push:
+    name: Push Images
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    defaults:
+      run:
+        working-directory: ./src/github.com/${{ github.repository }}
+    env:
+      GOPATH: ${{ github.workspace }}
```

Comment:
Shouldn't be any need to `cd` into the `GOPATH`, since this job doesn't run `go`.

Reply:
Actually the opposite, it needs the go action so the theoretical push command can build the images. Im going to remove the file so this PR can get approved some day

Reply:
The image we build now doesn't build any go... It just copies the base helm-operator image.

Reply:
example-apiserver has to build go code